retirement_api/utils/ss_utilities.py

```diff
@@ -123,21 +124,22 @@ def yob_test(yob=None):
     tests to make sure suppied birth year is valid;
     returns valid birth year as a string or None
     """
+    log = logging.getLogger(__name__)
     today = datetime.datetime.now().date()
     if not yob:
         return None
     try:
         birth_year = int(yob)
     except:
-        print "birth year should be a number"
+        log.warn("birth year should be a number")
```

Comment:
I had to choose log levels for all of the messages that were formerly being printed to the console. I chose `info` for most of them, but some sounded more like `warn`, such as these.